app/api/v1/models/questions_models.py

```diff
@@ -24,38 +26,24 @@ def post_question(self, postedBy, meetup_id, title, content):
         return new_question, {"message": "Question added successfully"}
 
     def getall_questions(self):
+        """method to return all questions"""
         return self.db
 
     def getone_question(self, question_id):
+        """method to get one question"""
         question = self.check_item(question_id, "question_id", questions)
         return question
 
-    def upvote_question(self, question_id):
-        """ method to upvote question """
-
-        question = self.check_item(question_id, "question_id", questions)
-        if question:
-            upvote = {
-                "postedBy": question[0]["postedBy"],
-                "meetup_id": question[0]["meetup_id"],
-                "title": question[0]["title"],
-                "content": question[0]["content"],
-                "votes": self.votes + 1
-            }
-
-            return upvote, {"message": "upvote successfull"}
-
-    def downvote_question(self, question_id):
-        """ method to downvote question """
-
-        question = self.check_item(question_id, "question_id", questions)
-        if question:
-            downvote = {
-                "postedBy": question[0]["postedBy"],
-                "meetup_id": question[0]["meetup_id"],
-                "title": question[0]["title"],
-                "content": question[0]["content"],
-                "votes": self.votes - 1
-            }
-
-            return downvote, {"message": "downvote successfull"}
+    def upvotes(self,question_id):
```

Comment:
The method upvotes() and downvotes() contain a duplicate implementation. I would suggest that you consider refactoring.

Reply:
@kwhilo, noted with thanks. I intend to add a function in the validation.py to counter the same as soon as this PR is approved.